conceptual.md

```diff
@@ -0,0 +1,49 @@
+### Conceptual Exercise
+
+Answer the following questions below:
+
+- What is React? When and why would you use it?
+    - React is a front-end frame work.
+    - When to use: when you are building an application that implements reusable components
+    - 	Why use: it can make creating apps much simpler
+
+- What is Babel?
+	- Babel is a JavaScript transpiler that converts alternate forms of JavaScript (e.g., JSX) into regular JavaScript.
```

Comment:
It's mainly used to make new Javascript language features backwards compatible with older versions.